### DISCUSSION TITLE
Single core verses multicore

### DISCUSSION THREAD

**Discussion by NathanTech7713** [lisdude/toaststunt#91] [Q&A]
Hey,

So in my head, true or otherwise, I have the notion that Lambda Moo when it was released, both due to the year of release and such, is designed for single core system and can not take advantage of multi-core CPU's, is that right?

to that end, is toaststunt still the same, limited to one core?

thanks

**Comment by tvdijen** [comment]
I think you mean single-process instead of single core or single CPU..
The amount of cores and CPU's should be irrelevant to a piece of software.

Software can be made specifically multi-threaded, but it's incredibly hard to do so. What's easier to do is to fork into multiple sub-processes and let the operating system decide what CPU/core to use. This is already happening in Lambdamoo during for example a database-dump.
I mean, if what you're asking is if MOO in general can be optimized then the answer is probably yes, but is it really worth the effort with nowadays CPU-power?
